plugins/octopus-deploy/api-report.md

```diff
@@ -22,11 +22,15 @@ export const isOctopusDeployAvailable: (entity: Entity) => boolean;
 // @public (undocumented)
 export const OCTOPUS_DEPLOY_PROJECT_ID_ANNOTATION = 'octopus.com/project-id';
 
+// @public (undocumented)
+export const OCTOPUS_DEPLOY_SPACE_ID_ANNOTATION = 'octopus.com/space-id';
+
 // @public (undocumented)
 export interface OctopusDeployApi {
   // (undocumented)
   getReleaseProgression(
     projectId: string,
+    spaceId: string | null,
     releaseHistoryCount: number,
   ): Promise<OctopusProgression>;
```

Comment:
I think now would be a good time to change this API to use an object instead whilst we're breaking the API anyways, and that way it can be evolved over time much easier.

```suggestion
  getReleaseProgression(opts: {
    projectId: string,
    spaceId?: string,
    releaseHistoryCount: number,
  }): Promise<OctopusProgression>;
```